introduction.md

```diff
@@ -1 +1,5 @@
-Hi I am Teejay. Favorite code school path: HTML.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?